system/jlib/jthread.cpp

```diff
@@ -1003,8 +1019,9 @@ class CThreadPool: public CThreadPoolBase, implements IThreadPool, public CInter
     PooledThreadHandle _start(void *param,const char *name, bool noBlock, unsigned timeout=0)
     {
         CCycleTimer startTimer;
-        bool waited = false;
-        bool timedout = false;
+        bool waited{false};
```

Comment:
@jakesmith thoughts on using this style for local variables - generally we haven't used it for local variables, except possibly for initialising structures.

Reply:
yes, I pondered about whether I liked it or not.. 
I concluded that it is quite common modern practice, and as long as consistent (in method or in class member initialization), I'm okay with it.